static/js/invite.js

```diff
@@ -62,23 +61,21 @@ function submit_invitation_form() {
             var arr = JSON.parse(xhr.responseText);
             if (arr.errors === undefined) {
                 // There was a fatal error, no partial processing occurred.
-                invite_status.text(arr.msg)
-                    .addClass('alert-error')
-                    .show();
+                ui_report.error("", xhr, invite_status);
             } else {
                 // Some users were not invited.
                 var invitee_emails_errored = [];
-                var error_list = $('<ul>');
-                _.each(arr.errors, function (value) {
-                    error_list.append($('<li>').text(value.join(': ')));
+                var error_list = [];
+                arr.errors.forEach(function (value) {
```

Comment:
Why the change from `_.each` to `forEach`?

Reply:
No, no any particular reason, it's just I rewrite this part so used native `forEach` than `_.each`